service/github/github.go

```diff
@@ -300,12 +301,17 @@ func (g *PullRequest) setPostedComment(ctx context.Context) error {
 
 // Diff returns a diff of PullRequest.
 func (g *PullRequest) Diff(ctx context.Context) ([]byte, error) {
-	opt := github.RawOptions{Type: github.Diff}
-	d, _, err := g.cli.PullRequests.GetRaw(ctx, g.owner, g.repo, g.pr, opt)
+	pr, _, err := g.cli.PullRequests.Get(ctx, g.owner, g.repo, g.pr)
 	if err != nil {
 		return nil, err
 	}
-	return []byte(d), nil
+
+	bytes, err := exec.Command("git", "diff", "--find-renames", pr.GetBase().GetSHA(), pr.GetHead().GetSHA()).CombinedOutput()
```

Comment:
We need to compare to merge base of SHA to get the same diff as Pull request.

Example: https://github.com/reviewdog/reviewdog/blob/dd1167b11c4d5eee154151ac1626afe24fd7cb95/service/gitlab/gitlab_mr_diff.go#L63-L65

Reply:
I apply it: https://github.com/reviewdog/reviewdog/pull/1697/commits/3c496f50759521c76f0d43f38d0eefaebf8819e9